Ginger/DatabaseUnitTest/UnitTest1.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace MSdatabaseUT
+{
+    [TestClass]
+    public class UnitTest1
```

Comment:
remove this class or change name to meaningful

Reply:
done